qutip/solve/steadystate.py

```diff
@@ -363,7 +375,8 @@ def steadystate(A, c_op_list=[], method='direct', solver=None, **kwargs):
 
     # Set weight parameter to max abs val in L if not set explicitly
     if 'weight' not in kwargs.keys():
-        ss_args['weight'] = np.abs(_data.norm.max(A.data))
+        # set the weight to the mean of the non-zero absoluate values in A:
+        ss_args['weight'] = np.mean(np.abs(A.full().data))
```

Comment:
This change is the source of two errors related to the iterative methods not converging. @Ericgig why is this change necessary?

Reply:
This was changed in #1802.